nixos/modules/services/backup/borgbackup.nix

```diff
@@ -695,6 +718,10 @@ in {
         # A repo named "foo" is mapped to systemd.services.borgbackup-repo-foo
         // mapAttrs' mkRepoService repos;
 
+      # A job named "foo" is mapped to systemd.timers.borgbackup-job-foo
+      # only generate the timer if interval (startAt) is set
+      systemd.timers = mapAttrs' mkBackupTimers (filterAttrs (cfg: cfg.startAt != []) jobs);
```

Comment:
```suggestion
      systemd.timers = mapAttrs' mkBackupTimers (filterAttrs (_: cfg: cfg.startAt != []) jobs);
```

Reply:
ignore the first suggestion, put the argument in the wrong place by accident :sweat:

Reply:
still does not work :disappointed:

Reply:
we mistyped the suggestion at first, looks like you applied that wrong one. we tested locally with this one and it worked out :)